modules/oe_theme_helper/src/TwigExtension/Filters.php

```diff
@@ -61,16 +65,22 @@ public function toLanguageName($language_code): string {
    *
    * @return string
    *   The native language name.
-   *
-   * @throws \InvalidArgumentException
-   *   Thrown when the passed in language code does not exist.
    */
-  public function toNativeLanguageName($language_code): string {
+  public function toNativeLanguageName($language_code) {
+    if (empty($language_code)) {
+      return $this->t('Unknown');
+    }
+
+    if ($language_code == LanguageInterface::LANGCODE_NOT_SPECIFIED) {
+      return $this->t('None');
```

Comment:
Don't forget to declare the return type of the function.

Reply:
Updated to `@return string|\Drupal\Core\StringTranslation\TranslatableMarkup`